hardware/computemodule/cm-emmc-flashing.md

```diff
@@ -118,25 +118,36 @@ The `/dev/sdX1` and `/dev/sdX2` partitions can now be mounted normally.
 
 Make sure J4 (USB SLAVE BOOT ENABLE) / J2 (nRPI_BOOT) is set to the disabled position and/or nothing is plugged into the USB slave port. Power cycling the IO board should now result in the Compute Module booting from eMMC.
 
-## Flashing the bootloader EEPROM - Compute Module 4
-The `rpiboot` tool is the recommended method for updating the bootloader EEPROM on Compute Module 4. After following the initial EMMC flashing setup steps run the following command to run the `recovery` image instead of the EMMC image.
+<a name="cm4bootloader"></a>
+## Compute Module 4 bootloader
+The default bootloader configuration on CM4 is designed to support bringup and development on a [Compute Module 4 IO board](https://www.raspberrypi.org/products/compute-module-4-io-board) and the software version flashed at manufacture may be older than the latest release. For final products please consider:-
 
-```bash
-./rpiboot -d recovery
-```
+* Selecting and verifying a specific bootloader release. The version in the `usbboot` repo is always a recent stable release.
+* Configuring the boot device (e.g. network boot). See `BOOT_ORDER` section in the [bootloader configuration](raspberrypi/bcm2711_bootloader_config.md) guide.
+* Enabling hardware write protection on the bootloader EEPROM to ensure that the bootloader can't be modified on remote/inaccessible products.
 
-The `recovery` directory of the `rpiboot` tool contains a default `pieeprom.bin` bootloader EEPROM image. See the [boot EEPROM](../raspberrypi/booteeprom.md) and [bootloader configuration](../raspberrypi/bcm2711_bootloader_config.md) pages for more information about how to change the embedded configuration file.
+N.B. The Compute Module 4 ROM never runs `recovery.bin` from SD/EMMC and the `rpi-eeprom-update` services is not enabled. This is necessary because the EMMC is not removable and an invalid `recovery.bin` file would prevent the system from booting.
```

Comment:
"services is not enabled" should be "services are not enabled"?

Reply:
Should be "service" singular